features/admin.server-configurations.v1/pages/account-disable-page.tsx

```diff
@@ -189,8 +197,18 @@ export const AccountDisablePage: FC<AccountDisablePageInterface> = (
                 "accountDisableHandler.friendlyName") }
             pageTitle={ t("governanceConnectors:connectorCategories.accountManagement.connectors." +
                 "accountDisableHandler.friendlyName") }
-            description={ t("governanceConnectors:connectorCategories.accountManagement.connectors." +
-                "accountDisableHandler.description") }
+            description={ (
+                <>
+                    { t("governanceConnectors:connectorCategories.accountManagement.connectors." +
+                        "accountDisableHandler.description") }
+                    <DocumentationLink
+                        link={ getLink("manage.accountDisable.learnMore") }
+                    >
+                        { t("common:learnMore") }
+                    </DocumentationLink>
+                </>
+            )
+            }
```

Comment:
Formatting suggestion;
```suggestion
            description={
                (<>
                    { t("governanceConnectors:connectorCategories.accountManagement.connectors." +
                        "accountDisableHandler.description") }
                    <DocumentationLink
                        link={ getLink("manage.accountDisable.learnMore") }
                    >
                        { t("common:learnMore") }
                    </DocumentationLink>
                </>)
            }
```
We can reduce a unnecessary line with this.

Reply:
Fixed with 94f0ac32c3575ebf73ee6d2ad623ff8395d4bfe1